### PR TITLE
login form controls modified + responsive + create_account design change

### PR DESCRIPTION
**Pull request by Ada-Lungu**

### CODE REVIEW

File: zeeguu/static/css/create_account.sass [hunk 1]
```diff
@@ -3,27 +3,31 @@
...
-  font-size: 38px
+  font-size: 36px
 
 
-.inner-center 
+.inner-center
   display: block
   margin: 0px auto
   padding-top: 20px
   padding-bottom: 20px
```
Comment (mircealungu) [inline zeeguu/static/css/create_account.sass]: does the padding really belong into the .inner-center class? 
I would argue that you want a separate class for that. 
debate?

File: zeeguu/static/css/create_account.sass [hunk 2]
```diff
@@ -3,27 +3,31 @@
...
 
-.select2-active p
-  font-size: 40px
-  font-weight: bold
-
-
-.create-account-block-width 
+.create-account-block-width
   width: 500px
   background-color: rgb(244,244,244)
```
Comment (mircealungu) [inline zeeguu/static/css/create_account.sass]: the class is named block-width. why does it have colors inside it?

File: zeeguu/static/css/login.sass [hunk 3]
```diff
@@ -1,44 +1,77 @@
...
+.inner-center
+  /* Centering a div inside another div */
+  display: block
   margin-left: auto
   margin-right: auto
-  border: 1px solid #DEDEDE
-  padding: 15px
+  margin-top: 100px
+  padding-top: 20px
+  padding-bottom: 20px
```
Comment (mircealungu) [inline zeeguu/static/css/login.sass]: same observation as before -- .inner-center should only do the centering business, no padding needed for that!

File: zeeguu/templates/create_account.html [hunk 4]
```diff
@@ -9,23 +9,26 @@
...
 
                     <div class="input-group-lg">
-                      <label>Email:</label>
+                      <label>Email</label>
                       <input type="email" name="email" id="email" placeholder="Email address" class="form-control input-lg" rel="popover" />
                     </div>
 
                     <div class="input-group-lg">
-                      <label class="control-label">Choose a Password:</label>
+                      <label class="control-label">Choose a Password</label>
```
Comment (mircealungu) [inline zeeguu/templates/create_account.html]: Removing the ":". Nice attention to detail :)

File: zeeguu/static/css/create_account.sass (hunk 1 above)
Comment (Ada-Lungu) [inline zeeguu/static/css/create_account.sass]: Right. Moved them to .create-account-block-width, i mean padding e ok sa am aici in width, nu?;p

**Pull request merged**